docs/source/markdown/podman-system.1.md

```diff
@@ -15,9 +15,10 @@ The system command allows you to manage the podman systems
 | -------  | --------------------------------------------------- | ---------------------------------------------------------------------------- |
 | df       | [podman-system-df(1)](podman-system-df.1.md)        | Show podman disk usage.                                                      |
 | info     | [podman-system-info(1)](podman-info.1.md)           | Displays Podman related system information.                                  |
+| migrate  | [podman-system-migrate(1)](podman-system-migrate.1.md)| Migrate existing containers to a new podman version.                       |
 | prune    | [podman-system-prune(1)](podman-system-prune.1.md)  | Remove all unused container, image and volume data                              |
 | renumber | [podman-system-renumber(1)](podman-system-renumber.1.md)| Migrate lock numbers to handle a change in maximum number of locks.      |
-| migrate  | [podman-system-migrate(1)](podman-system-migrate.1.md)| Migrate existing containers to a new podman version.                       |
+| reset    | [podman-system-reset(1)](podman-system-reset.1.md)| Reset storage back to original state.  Remove all pods, containers, images, volumes |
 
 ## SEE ALSO
```

Comment:
probably should add a line to commands.md too.

Reply:
Added, and also found that commands.md was broken. So added a patch to fix the links.

Reply:
Yeah, #4561 was working on fixing the links there, but it doesn't look like they've come back to sign their pr.  Thanks for touching those up.